webui/react/src/hooks/useCreateExperimentModal.tsx

```diff
@@ -41,6 +41,8 @@ interface ModalHooks {
   showModal: (props: ShowProps) => void;
 }
 
+const PADDING = 8;
```

Comment:
> > check that the log preview does not appear unless it has logs to display
> 
> suggestion: wdy think about excluding log entries that have an empty message as well? this might make it difficult tho. Would you repeatedly go back until there is an entry that has it? 🤔

Does that happen?! I didn't know we stored empty message log entries in the db...

Reply:
For `PADDING` and other shared config editor, I ended up extracting monaco-editor into a component and extending the `react-monaco-editor`. I wanted to apply `React.lazy` to this component as well, but I ran into the issue of the typescript types not being available due to the lazy loading. If we import the types only separate from the module, it still includes it into the main bundle (even though we only need the types).